deps.edn

```diff
@@ -4,8 +4,7 @@
         cheshire/cheshire {:mvn/version "5.11.0"}
         nrepl/bencode {:mvn/version "1.1.0"}
         org.babashka/sci {:mvn/version "0.7.38"}
-        babashka/fs {:mvn/version "0.2.16"}
```

Comment:
This probably explains the issue with the pod test.

Reply:
my bad, fixing it